### ISSUE TITLE
Typo in priorityDataClasses?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blurts-server#1408]
**TL;DR:** "phsyical-addresses" should be "physical-addresses"?

---

Found while randomly reviewing priority data classes to see if I could tie them back to specific breaches:

```js
const priorityDataClasses = {
  "government-issued-ids" : {
    weight: 101,
    pathToGlyph: "svg/glyphs/social-security-numbers",
  },
  "social-security-numbers" : {
    weight: 101,
    pathToGlyph: "svg/glyphs/social-security-numbers",
  },
  "passwords": {
    weight: 100,
    pathToGlyph: "svg/glyphs/passwords",
    actions: "", // future functionality?
  },
  "bank-account-numbers": {
    weight: 99,
    pathToGlyph: "svg/glyphs/bank-account-numbers",
  },
  "credit-cards": {
    weight: 98,
    pathToGlyph: "svg/glyphs/credit-cards",
  },
  "credit-card-cvv": {
    weight: 97,
    pathToGlyph: "svg/glyphs/credit-card-cvvs",
  },
  "partial-credit-card-data": {
    weight: 96,
    pathToGlyph: "svg/glyphs/partial-credit-card-data",
  },
  "ip-addresses": {
    weight: 95,
    pathToGlyph: "svg/glyphs/ip-addresses",
  },
  "historical-passwords": {
    weight: 94,
    pathToGlyph: "svg/glyphs/historical-passwords",
  },
  "security-questions-and-answers": {
    weight: 93,
    pathToGlyph: "svg/glyphs/security-questions-and-answers",
  },
  "phone-numbers": {
    weight: 92,
    pathToGlyph: "svg/glyphs/phone-numbers",
  },
  "email-addresses": {
    weight: 91,
    pathToGlyph: "svg/glyphs/email-addresses",
  },
  "dates-of-birth": {
    weight: 90,
    pathToGlyph: "svg/glyphs/dates-of-birth",
  },
  "pins": {
    weight: 89,
    pathToGlyph: "svg/glyphs/pins",
  },
  "phsyical-addresses": {
    weight: 88,
    pathToGlyph: "svg/glyphs/physical-addresses",
  },
};

for (const [key, value] of Object.entries(priorityDataClasses)) {
  if (!value.pathToGlyph.endsWith(`/${key}`)) {
    console.error(`"${key}" does not match ${value.pathToGlyph}`);
  }
}
```

### OUTPUT

```sh
node priority-test.js

"government-issued-ids" does not math svg/glyphs/social-security-numbers
"credit-card-cvv" does not math svg/glyphs/credit-card-cvvs
"phsyical-addresses" does not math svg/glyphs/physical-addresses
```

1. The **"government-issued-ids"** isn't very interesting. Looks like we're just aliasing some other partial, which is :+1:

2. The **"credit-card-cvv"** also isn't interesting since it looks like just a singluar-vs-plural difference between the dataClass and SVG/HBS file, which isn't an issue.

3. The **"phsyical-addresses"** looks like a typo. Looking at the /hibp/breaches JSON blob, there aren't any matching typos in the `breach.DataClasses` arrays, and if we look at a breach with a "_physical_-addresses" such as <https://monitor.firefox.com/breach-details/CafePress>, it seems like it isn't treating "Physical addresses" as a priority breach detail.

**Comment by pdehaan** [comment]
Bonus points if we can export the `priorityDataClasses` variable to make it easier to test/script. I think we've had other issues in the past where we were linking to a missing .hbs template in these priority data classes and it was throwing an error.

**Comment by pdehaan** [comment]
Updated the linter to also check for missing .hbs files and verify the `key` matches the `pathToGlyph`:

```js
"use strict";

const assert = require("assert");
const fs = require("fs");
const path = require("path");

// This works now that we're exporting the `priorityDataClasses` const.
const { priorityDataClasses } = require("./template-helpers/breach-detail");

Object.entries(priorityDataClasses).forEach(([dataClass, value]) => {
  try {
    const hbsPath = path.join("./views/partials", `${value.pathToGlyph}.hbs`);
    if (!fs.existsSync(hbsPath)) {
      console.log(`${value.pathToGlyph} not found at ${hbsPath}`);
    }

    const hbsGlyph = `svg/glyphs/${dataClass}`;
    assert.strictEqual(value.pathToGlyph, hbsGlyph);
  } catch (err) {
    console.error(err.message);
    process.exitCode = 1;
  }
});
```

### OUTPUT

```sh
node priorityDataClassesTest.js

Expected values to be strictly equal:
+ actual - expected

+ 'svg/glyphs/social-security-numbers'
- 'svg/glyphs/government-issued-ids'
              ^
Expected values to be strictly equal:
+ actual - expected

+ 'svg/glyphs/credit-card-cvvs'
- 'svg/glyphs/credit-card-cvv'
                             ^
```
